docs/en/08.support-for-snapdeal.md

```diff
@@ -0,0 +1,123 @@
+2022-04-15T04:28:13.955Z        DEBUG   policy  policy/const.go:18      NETWORK SETUP   {"name": "butterfly"}
+deadline  count
+3         2
+```
+As seen, there are currently 2 `CC sectors` available in the `#3 deadline` as candidates for upgrade.
+
+### Configuring damocles-worker
+In `damocles-worker`, the main configurations required are related to `SnapUp` job, such as `sealing_thread`s, and resource allocation for `snap_encode` and `snap_prove` tasks.
+
+Here is an example configuration:
+```toml
+[[sealing_thread]]
+location = "/data/local-snap-1"
+plan = "snapup"
+
+[[sealing_thread]]
+location = "/data/local-snap-2"
+plan = "snapup"
+
+[[sealing_thread]]
+location = "/data/local-snap-3"
+plan = "snapup"
+
+[[sealing_thread]]
+location = "/data/local-snap-4"
+plan = "snapup"
+
+[[sealing_thread]]
+location = "/data/local-snap-5"
+plan = "snapup"
+
+[[sealing_thread]]
+location = "/data/local-snap-6"
+plan = "snapup"
+
+[[sealing_thread]]
+location = "/data/local-snap-6"
+plan = "snapup"
+
+[processors.limitation.concurrent]
+# ...
+tree_d = 1
+snap_encode = 5
+snap_prove = 2
+
+[[processors.snap_encode]]
+envs = { FIL_PROOFS_USE_GPU_COLUMN_BUILDER = "1", FIL_PROOFS_USE_GPU_TREE_BUILDER = "1", CUDA_VISIBLE_DEVICES = "0", TMPDIR="/var/tmp/worker0" }
+
+[[processors.snap_prove]]
+envs = { CUDA_VISIBLE_DEVICES = "0" , TMPDIR="/var/tmp/worker0" }
+
+[[processors.snap_encode]]
+envs = { FIL_PROOFS_USE_GPU_COLUMN_BUILDER = "1", FIL_PROOFS_USE_GPU_TREE_BUILDER = "1", CUDA_VISIBLE_DEVICES = "0", TMPDIR="/var/tmp/worker1" }
+
+[[processors.snap_prove]]
+envs = { CUDA_VISIBLE_DEVICES = "1", TMPDIR="/var/tmp/worker1" }
+```
+
+The resource allocation for `snap_encode` can be similar to `pc2`, and for `snap_prove`, it can be similar to `c2`.
+
+### Configuring damocles-manager
+In `damocles-manager`, the required configuration mainly involves enabling `SnapUp` for specific `SPs`. Here is an example:
+
+```
+[[Miners]]
+Actor = 1153
+[Miners.Sector]
+InitNumber = 0
+MaxNumber = 10000
+Enabled = true
+EnableDeals = false
+
+[Miners.SnapUp]
+Enabled = true
+Sender = "t1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za"
+```
+
+In this configuration:
+- The settings in `[Miners.Sector]` do notSpecify the complete configuration details for the SnapUp production scheme. However, the key points to note are:
+- Set `Enabled = true` to enable SnapUp for the specified `SP`.
+- Specify the `Sender` address, which will be used to send SnapUp messages.
+
+### Starting the Production
+Once the configurations are in place, you can start the production process by running `damocles-manager` and `damocles-worker` with the appropriate configurations. The `damocles-worker` will automatically convert the `sealing_thread`s into `SnapUp` production plans and start processing the candidate sectors.
+
+Please note that the provided example is a simplified version, and the actual configuration and setup may vary based on your specific requirements and environment.
```

Comment:
```suggestion
- Considering the deployment of sector persistent storage, the `damocles-worker` used for `SnapUp` needs to be able to have both read and write access to all persistent storage spaces (persist stores), and ensure that their names are consistent with those defined in `damocles-manager`.
- Based on the above two points, we recommend using a separate box specifically for `SnapUp` production to avoid the configuration, operation and maintenance complexity caused by mixing regular sectors and `SnapUp` sealing.
```